tools/server-admin/build.gradle.kts

```diff
@@ -185,6 +185,9 @@ if (Os.isFamily(Os.FAMILY_MAC) && System.getenv("CI") != null) {
 }
 
 tasks.named<Test>("intTest").configure {
+  // Reduce likelihood of OOM due to too many Quarkus apps in memory;
+  // Ideally, set this number to the number of IT classes to run for each backend.
+  forkEvery = 5
```

Comment:
In fact, I think it's not a good idea to use MultiEnv with Quarkus. We should probably look into removing MultiEnv from this module's tests.